core/lib/zksync_core/src/consensus/storage/testonly.rs

```diff
@@ -0,0 +1,47 @@
+//! Storage test helpers.
+use anyhow::Context as _;
+use zksync_concurrency::{ctx, error::Wrap as _, time};
+use zksync_consensus_roles::validator;
+use zksync_consensus_storage as storage;
+
+use super::{BlockStore, CtxStorage};
+
+impl BlockStore {
+    /// Waits for the `want` miniblock to have a certificate.
```

Comment:
Nit: "`want` miniblock" -> "miniblock with the given `number`"?

Reply:
done